script/statistics

```diff
@@ -23,11 +29,11 @@ begin
     relation_count = Relation.where(:visible => true).count
 
     puts "<table>"
-    puts "<tr><td>Number of users</td><td>#{user_count}</td></tr>"
-    puts "<tr><td>Number of uploaded GPS points</td><td>#{tracepoint_count}</td></tr>"
-    puts "<tr><td>Number of nodes</td><td>#{node_count}</td></tr>"
-    puts "<tr><td>Number of ways</td><td>#{way_count}</td></tr>"
-    puts "<tr><td>Number of relations</td><td>#{relation_count}</td></tr>"
+    puts "<tr><td>Number of users</td><td>#{user_count.format()}</td></tr>"
+    puts "<tr><td>Number of uploaded GPS points</td><td>#{tracepoint_count.format()}</td></tr>"
+    puts "<tr><td>Number of nodes</td><td>#{node_count.format()}</td></tr>"
+    puts "<tr><td>Number of ways</td><td>#{way_count.format()}</td></tr>"
+    puts "<tr><td>Number of relations</td><td>#{relation_count.format()}</td></tr>"
```

Comment:
If you’re open to it, an alternative approach would be to add a small amount of JavaScript that [formats the number](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString) on the fly.

Reply:
I think that would be a very bad idea...